python-lib/tableau_server_utils.py

```diff
@@ -0,0 +1,46 @@
+"""
+Gives functions for interaction with Tableau Server through the Tableau Server Client API
+"""
+
+import logging
+
+import tableauserverclient as tsc
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format='Plugin: Tableau Hyper API | %(levelname)s - %(message)s')
+
+
+def get_project_from_name(server, project_name):
+    """
+    Retrieve the target project from Tableau Server
+    :param server: tableau server
+    :param project_name: target project name
+    :return: couple (Boolean{target project exists on server}, project)
+    """
+    page_nb = 1
+    while True:
```

Comment:
I don't like while True, don't we have an other way to iterate over the projects?

Reply:
Changed using the pagination info